rpc/connector/connector.proto

```diff
@@ -193,6 +196,16 @@ message IngressInfos {
   repeated manager.IngressInfo ingress_infos = 1;
 }
 
+message InstallRequest {
+  ConnectRequest connect_request = 1;
+  bool upgrade = 2;
+  repeated string value_paths = 3;
+}
+
+message InstallResult {
+  string error_text = 1;
```

Comment:
This should include an `error_category` and be handled similar to the current `UninstallResult` to avoid users being prompted to seek further information in the logs when entering flags etc. incorrectly.